src/wasm/kotlin/src/arcs/Particle.kt

```diff
@@ -69,8 +69,9 @@ abstract class Particle : WasmObject() {
         )
     }
 
-    open fun fireEvent(slotName: String, eventName: String) {
-        eventHandlers[eventName]?.invoke()
+    open fun fireEvent(slotName: String, eventName: String, eventData: Map<String, String>) {
+        log("particle.kt ${eventData}")
```

Comment:
Probably not good to log the event data itself, in case it contains sensitive information... maybe just the event name?

Reply:
Removed log line - was mostly there for initial testing purposes.